tests/plugins/forms/manual/required.html

```diff
@@ -0,0 +1,15 @@
+<body>
+<div id="editor">
+	<p>
+		<input required type="checkbox">
```

Comment:
I'd be nice to have more than one `[required]` form, e.g. `[required=whatever]`, `[required=false]` etc.